avro-serializer/src/main/java/io/confluent/kafka/serializers/AbstractKafkaAvroSerDe.java

```diff
@@ -35,6 +34,7 @@
   protected static final String SCHEMA_REGISTRY_URL = "schema.registry.url";
   protected static final String MAX_SCHEMAS_PER_SUBJECT = "max.schemas.per.subject";
   protected static final String SPECIFIC_AVRO_READER = "specific.avro.reader";
+  protected static final String GENERIC_AVRO_DATUM_WRITER_CLASS = "generic.avro.writer.class";
```

Comment:
Just a heads up that #162 is moving these settings around, so this patch will probably need to be updated to match. That said, #162 should avoid the duplicate options parsing logic, so hopefully it should be an improvement.